### ISSUE TITLE
Velocity®2X [PCSE00374]

### ISSUE THREAD

**Issue by MontyJack2020** [Vita3K/compatibility#1469]
# App summary
- App name: Velocity®2X
- App serial: PCSE00374
- App version: 1.00

# Vita3K summary
- Version: v0.1.8
- Build number: 3157
- Commit hash: https://github.com/vita3k/vita3k/commit/[4280bea5]
- CPU backend: Dynarmic
- GPU backend: Vulkan

# Test environment summary
- Tested by: JackMonty
- OS: Windows 10
- CPU: AMD Ryzen 5 Pro 3500U
- GPU: Nvidia Geforce GT1030
- RAM: 16 GB

# Issues
Game runs fine with good speed and audio. Comes up with the failed to sign in screen on first boot but once OK is pressed it goes away. There is some slight graphical corruption for certain affects when using Vulkan - these are much lessened when using OpenGL but the game runs slower on this - best to use Vulkan the minor graphics glitches don't hinder game-play.

# Screenshots

Title Screen
![Velocity1](https://github.com/Vita3K/compatibility/assets/130913305/c1e9290a-ac7e-495a-91e8-adecbd762afa)

Gameplay1
![Velocity2](https://github.com/Vita3K/compatibility/assets/130913305/fe4ab8c2-ddd6-4ba7-a568-073f2e923e6a)

Gameplay2
![Velocity3](https://github.com/Vita3K/compatibility/assets/130913305/b3407784-821a-4d26-a382-dc8e7c6a758f)

Graphics Glitch on Vulkan
![Velocity4](https://github.com/Vita3K/compatibility/assets/130913305/1803b6c1-3982-4a60-865e-affb47217540)

Graphics Glitch on Vulkan
![Velocity5](https://github.com/Vita3K/compatibility/assets/130913305/74dda236-4465-4c70-b153-4b2d628d5a5e)

Graphics Glitch on OpenGL
![Velocity6](https://github.com/Vita3K/compatibility/assets/130913305/e5e0cf74-15eb-4905-afed-368bd3fa8e2d)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/11471365/vita3k.log)

# Recommended labels

- Playable
- Graphics Bug

**Comment by legolas119** [comment]
hi, i tried several times to install it, but during the installation process, Vita3k immediately crashes. So i copy directly the game folder into the ux0\app path, but when i run the game, it appears this error message:
https://i.postimg.cc/Sx1MHzYt/velocity.png

Do you know why? I'm using the latest vita3k version

thank you

**Comment by Soyboy-69** [comment]
I have the same issue. Vita3k crashes right before the installation is finished.

Not sure how to fix this issue.

**Comment by haxwagon** [comment]
@Soyboy-69 , same issue here.  Have you found any logs?  Maybe we could see what the issue is between the two of us.  Are you testing this on Android?

**Comment by Soyboy-69** [comment]
> @Soyboy-69 , same issue here. Have you found any logs? Maybe we could see what the issue is between the two of us. Are you testing this on Android?

No, I have not found any logs... And yeah, I'm testing it on Android.
